docs/Versioning.md

```diff
@@ -0,0 +1,73 @@
+# ML-Agents Versioning
```

Comment:
Can we add a subsection that describes our GitHub Releases and the numbering scheme.